src/set_theory/cardinal/basic.lean

```diff
@@ -612,6 +612,11 @@ induction_on a $ λ α, mk_congr $
 
 theorem sum_const' (ι : Type u) (a : cardinal.{u}) : sum (λ _:ι, a) = #ι * a := by simp
 
+@[simp] theorem sum_add_distrib {ι} (f g : ι → cardinal) :
+  sum (f + g) = sum f + sum g :=
+by simpa only [mk_sigma, mk_sum, mk_out, lift_id] using
+  mk_congr (equiv.sigma_sum_distrib (quotient.out ∘ f) (quotient.out ∘ g))
```

Comment:
Rather than using `quotient.out` which invoke choice, I think you should be using induction on `f` and `g`; or does that not work so well?

Reply:
`quotient.induction_on_pi` seems to work

Reply:
Ah sorry, I missed these were pi types. Your original version was probably better